.changeset/eleven-poems-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@smithy/util-waiter": minor
```

Comment:
This is a fix and there is no API addition, so the changeset type should be "patch".